demos/src/accordion.mustache

```diff
@@ -0,0 +1,38 @@
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="3" data-o-expander-count-selector="li">
```

Comment:
I added the simple expander just to ensure that both of them work fine if they are initialised at the same time, but that's probably more a test-thing than a demo-thing